### ISSUE TITLE
Cannot unassign properly from record audit page

### ISSUE THREAD

**Issue by SyedaAfrida** [opencrvs/opencrvs-core#6118]
**Bug description:**
Cannot unassign properly from record audit page

**Steps to reproduce:**
1. Login to Registrar
2. Go to the record audit page of a record which is assigned to someone else
3. Try to unassign the record
4. Observe

**Actual result:**
Cannot unassign properly from record audit page. But when clicked back it is shown as unassigned

**Expected result:**
User should be able to unassign properly

**Screen recording:**

https://github.com/opencrvs/opencrvs-core/assets/94350336/48d03231-d730-488c-a3c0-68245199eddd

**Tested on:**
https://login.farajaland-qa.opencrvs.org/?lang=en

**Comment by tahmidrahman-dsi** [comment]
Could not reproduce in QA env

https://github.com/opencrvs/opencrvs-core/assets/42269993/85404c22-6da3-43fc-a715-93c51a6d1fa0

**Comment by SyedaAfrida** [comment]
@tahmidrahman-dsi Earlier we used to have this bug in record audit page after searching. Currently after all new deployments this is now fixed in record audit page but appearing in workqueue. However, the bug is completely the same.

**Steps to reproduce:** 
1. Go to advance search
2. Search any birth/ death declaration by location and status
3. From workqueue try to unassign other user from any record
4. Observe


https://github.com/opencrvs/opencrvs-core/assets/94350336/ca9e3988-d14b-4bde-92bd-24141a03ea3c
